.travis.yml

```diff
@@ -38,6 +40,9 @@ jobs:
   allow_failures:
   - python: nightly
   include:
+  - python: 3.7
```

Comment:
I'd say it's better to use `3.7-dev` so we don't require `sudo` nor run on a different distribution.
I don't know why travis 3.7 has problems, but using `3.7-dev` works fine.

Reply:
3.7-dev is outdated, has different source and probably has something like openssl disabled. I'd insist on 3.7. As for different env for wheels for publishing to PYPI, those are being built in manylinux1, which is still the same as before.

Reply:
It doesn't bother me for the wheels, those don't depend on it as they use the python versions provided by the docket image, but it makes me feel quite unease to run the tests on different systems, I dislike inconsistencies :P
Also `sudo: true` means our builds will be queued longer as they will require a full virtual machine instead of running in a container.

Btw I'll merge it as it, I don't like to promote bikeshedding.

Reply:
I agree with the point that having different envs is unfortunate, but now it's the best we can do.
Also, turns out that sometimes the end-result of having VM vs. containter-based worker is practically about the same and in some cases VM-based tests might go even faster, because while it has slower startup time runtime tends to have better access to resources. It's from experience, before I compared it myself I also thought that docker-based jobs are faster in 100% of cases, but turned out that it's not always true.